db/migrate/20211119104835_add_property_info_fields.rb

```diff
@@ -0,0 +1,11 @@
+class AddPropertyInfoFields < ActiveRecord::Migration[6.1]
+  def change
+    change_table :case_logs, bulk: true do |t|
+      t.column :first_time_property_let_as_social_housing, :int
+      t.column :why_dont_you_know_la, :string
+      t.column :type_property_most_recently_let_as, :string
+      t.column :builtype, :string
+      t.rename :tenant_same_property_renewal, :renewal
```

Comment:
Renewal should also be an int/enum right?

Reply:
Both ints yes type_property_most_recently_let_as is UNITLETAS and changed to reflect